tests/integration/slf4j-hybrid-consumer/src/test/kotlin/aws/smithy/kotlin/test/LoggingSlf4jHybridTest.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.test
+
+import aws.smithy.kotlin.runtime.telemetry.logging.LogLevel
+import aws.smithy.kotlin.runtime.telemetry.logging.getLogger
+import aws.smithy.kotlin.runtime.telemetry.logging.slf4j.Slf4jLoggerProvider
+import aws.smithy.kotlin.runtime.telemetry.logging.warn
+import kotlin.test.Test
+
+class LoggingSlf4jHybridTest {
```

Comment:
**Comment**: Thanks for submitting a test project that confirms the correct behavior. This is great!

Reply:
Yeah this was 99% of the work